### PR TITLE
DDP-8603/OS-email-address-links (front-end)

### PR DESCRIPTION
**Pull request by llobzhanidze**
https://broadinstitute.atlassian.net/browse/DDP-8603
Add email links, also change links color like was in ticket

### CODE REVIEW

File: ddp-workspace/projects/ddp-osteo/src/styles.scss [hunk 1]
```diff
@@ -58,6 +58,10 @@ ul {
     margin: 0;
 }
 
+.maincolor {
```
Comment (MishaK1204) [inline ddp-workspace/projects/ddp-osteo/src/styles.scss]: Please write more descriptive class name, E.g: .emailColor;

File: ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json [hunk 2]
```diff
@@ -755,7 +755,7 @@
                                     "content": "“Are there any benefits that participants receive for participating?“"
                                 }
                             ],
-                            "To take part in these activities, please log into your dashboard to sign additional consent forms and provide updates to your/your child’s participation. If you have any questions, please reach out to the study team at info@osproject.org or 651-602-2020."
+                            "To take part in these activities, please log into your dashboard to sign additional consent forms and provide updates to your/your child’s participation. If you have any questions, please reach out to the study team at <a href=\"mailto:info@OSproject.org\" class='maincolor'>info@OSproject.org</a> or 651-602-2020."
```
Comment (MishaK1204) [inline ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json]: Is not anchor tag already blue?

**Review by GiCharkviani — CHANGES_REQUESTED** [review]
Could you please provide more details in the description.

File: ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json (hunk 2 above)
Comment (llobzhanidze) [reply ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json]: no it's not, this is a reason why I add class for color

File: ddp-workspace/projects/ddp-osteo/src/styles.scss (hunk 1 above)
Comment (llobzhanidze) [reply ddp-workspace/projects/ddp-osteo/src/styles.scss]: it's fixed

**Review by MishaK1204 — APPROVED** [review]

**Review by GiCharkviani — APPROVED** [review]

**Pull request merged**